doc/api/process.markdown

```diff
@@ -16,6 +16,12 @@ continue.
 as [`process.exit()`][] or uncaught exceptions, and should not be used as an
 alternative to the `'exit'` event unless the intention is to schedule more work.
 
+## Event: 'disconnect'
+
+If Node.js was spawned with an IPC channel, this event will be emitted when
+either [`process.disconnect()`][] in child process or
+[`ChildProcess.disconnect()`] in parent process is called.
```

Comment:
@sam-github Well, I just copied it from `child_process` doc I guess. Should I fix these lines to something like ".... in parent process is called or when one of the processes faces unrecoverable error (and terminates)?"